src/subscribr.js

```diff
@@ -55,6 +55,12 @@ module.exports = class Subscribr {
 
 }
 
+let idCounter = 0;
```

Comment:
I'd rather having 0 production dependencies, and seems that [the lodash uniqueId feature](https://github.com/lodash/lodash/blob/master/uniqueId.js) is pretty similar to this one